Documentation/Signoff/Logic-gate-subsystem.md

```diff
@@ -0,0 +1,102 @@
+# Logic Gates Subsystem
+
+##  Function of the System
+The function of this subsystem is to introduce the concept of logic gates through an approachable way for someone with little to no experience using AND,OR and NOT gate. The subsystem will be user-friendly by using switches and LEDs to demonstrate these concepts and allow the user to interact with the inputs.
+
+
+## Constraint
+
+| Description | Constraints                                                   | Origin            |
+| --- | --------------------------------------------------------------------- | -----------------|
+| Current regulation | Current in the system shall be between 15 mA and 30 mA [3] | LED Constraint |
+| Voltage regulation | System shall use 5v from the Arduino UNO to operate | Device Constraint |
+|  Explicit  | Switches shall be labeled for a user-friendly interaction  | simplicity Constraint|
+| Cost Limit  |Any components and equipment within this subsystem should not exceed $40 dollars and the subsystems total cost should not exceed $120 to help keep the project budget in the 400-900 range| Conceptual Design|
+
+## Buildable schematic
+
+![image](https://github.com/user-attachments/assets/726d0e72-cc31-455a-bc9a-9b4c88de9018)
+
+
```

Comment:
You need to add a PCB layout. If we build this all on one board (with other members of your team) you will be able to easily pull your design into the main board.